### DISCUSSION TITLE
Model weights

### DISCUSSION THREAD

**Discussion by sushilkhadkaanon** [Vision-CAIR/MiniGPT-4#351] [Q&A] [answered]
Model weight repo contains multiple bin files. such as pytorch_model-00001-of-00002.bin, pytorch_model-00002-of-00002.bin.
Am I supposed to download one of this file or the whole repo and set the path of whole repo?

**Comment by sushilkhadkaanon** [comment, marked as answer]
Update: I solved this issue. We need to download the whole repo and set the path of the whole repp.

**Reply by bx24** [reply]
怎么操作的 谢谢
